style.css

```diff
@@ -0,0 +1,119 @@
+* {
+  margin: 0%;
+  padding: 0%;
+}
+
+#navbar {
+  display: flex;
+  justify-content: flex-end;
+  top: 0;
+  left: 0;
+  background-color: rgb(201, 77, 97);
+  font-size: 20px;
+}
+
+.navlink {
+  margin-right: 15px;
+  color: #f0f0f0;
+  padding: 18px;
+}
+
+#welcome-section {
+  background-color: rgb(228, 210, 220);
+  height: 100vh;
+  width: auto;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  justify-content: center;
+}
+
+h1 {
+  margin: 30px;
+  font-size: 50px;
+  font-family: sans-serif;
+}
+
+h2 {
+  font-size: 30px;
+  font-family: sans-serif;
+}
+
```

Comment:
I guess you have `font-family: sans-serif;` repeated in different classes and IDs in your style.css file. I think you can make your CSS dry by putting the style in a body tag. That way, It will apply to all text on your website. 

You can do something like this 👇🏽 
``` 
body {
  font-family: sans-serif;
}
```
Then remove it from other places where they have been applied.